charts/rancher-logging/0.1.2/charts/fluentd/charts/fluentd-windows/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+name: fluentd-windows
+version: 0.0.1
+appVersion: 1.1.3
```

Comment:
These versions don't line up with the versions mentioned in the directory structure